cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/azure/azure-mc-common/AzureClusterSettingsStep.tsx

```diff
@@ -0,0 +1,142 @@
+// React imports
+import React, { useContext, useState } from 'react';
+
+// Library imports
+import { CdsButton } from '@cds/react/button';
+import { CdsIcon } from '@cds/react/icon';
+import { yupResolver } from '@hookform/resolvers/yup/dist/yup';
+import * as yup from 'yup';
+
+// App imports
+
+import { AZURE_FIELDS } from '../AzureManagementCluster.constants';
+import { AzureStore } from '../../../../state-management/stores/Azure.store';
+import { ClusterName, clusterNameValidation } from '../../../../shared/components/FormInputComponents/ClusterName/ClusterName';
+import { INPUT_CHANGE } from '../../../../state-management/actions/Form.actions';
+import {
+    NodeInstanceType,
+    nodeInstanceTypeValidation,
+    NodeProfile,
+} from '../../../../shared/components/FormInputComponents/NodeProfile/NodeProfile';
+import { StepProps } from '../../../../shared/components/wizard/Wizard';
+import { SubmitHandler, useForm } from 'react-hook-form';
+
+// NOTE: icons must be imported
```

Comment:
Shouldn't there be a `ClarityIcons.addIcons(blockIcon, blocksGroupIcon);`?

Reply:
I think what I'm noticing is that this only has to be done once in the app for each icon...which leads me to thinking that we should have an initIcons module that loads all of the icons we use instead of doing it spread out through different components. How does that sound?

Reply:
I like it